Canvas/SubmitAssignment/SubmitAssignmentPresenter.swift

```diff
@@ -122,9 +122,6 @@ class SubmitAssignmentPresenter {
                 .appendingPathComponent(url.lastPathComponent)
             do {
                 try url.move(to: newURL)
-                if let image = UIImage(contentsOfFile: newURL.path), let data = image.normalize().jpegData(compressionQuality: 0.8) {
```

Comment:
`UIImage.normalize()` uses too much memory for share extensions. We'll have to normalize on the backend or via PSPDFKit or something.